examples/grok-4-vs-gpt-5/promptfooconfig.yaml

```
@@ -0,0 +1,137 @@
+    **Task 7 — Factual Knowledge**  
+    Who owns **Grok‑4** and who developed **GPT‑5**? Provide company names only.
+
+    **Task 8 — Hallucination Test**  
+    Who is the **current CEO of NodeShift Cloud**?  
+    *(If unknown, answer: "I don't know.")*
+
+    **Task 9 — SQL Querying**  
+    Write a SQL query to find the **top 5 highest-paid employees** from a table `employees` with columns `(id, name, salary)`.
+
+    **Task 10 — JSON Transformation**  
+    Convert this CSV into **JSON**:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)